File: testdata/jennies/rawtypes/dashboard/PHPRawTypes/src/Types/Dashboard/FieldConfig.php
<?php

namespace Grafana\Foundation\Types\Dashboard;

class FieldConfig implements \JsonSerializable {
    public ?string $unit;

    /**
     * @var mixed
     */
    public $custom;

    /**
     * @return array<string, mixed>
     */
    public function jsonSerialize(): array
    {
        $data = [
        ];
        if (isset($this->unit)) {
            $data["unit"] = $this->unit;
        }
        if (isset($this->custom)) {
            $data["custom"] = $this->custom;
        }
        return $data;
    }
}

File: testdata/jennies/rawtypes/dashboard/CSharpRawTypes/src/Grafana/Foundation/Dashboard/FieldConfig.cs
namespace Grafana.Foundation.Dashboard;


public class FieldConfig
{
    public string Unit;
    public object Custom;

    public FieldConfig()
    {
    }

    public FieldConfig(string unit, object custom)
    {
        this.Unit = unit;
        this.Custom = custom;
    }
}
